### PR TITLE
[CS-3103]: Create new wallet screen

### PR DESCRIPTION
**Pull request by danibonilha**
### Description
This PR adds the new wallet screen into the tabBar navigation, also refactors the way the wallet is initialized into more granular functions 

### Screenshots

image

image

### CODE REVIEW

File: src/hooks/useWalletManager.ts [hunk 1]
```diff
@@ -20,128 +27,142 @@ import { checkPushPermissionAndRegisterToken } from '@cardstack/models/firebase'
 import { useLoadingOverlay } from '@cardstack/navigation';
 import { appStateUpdate } from '@cardstack/redux/appState';
 import { getCurrencyConversionsRates } from '@cardstack/services';
+import { saveAccountEmptyState } from '@rainbow-me/handlers/localstorage/accountLocal';
 import { setCurrencyConversionRates } from '@rainbow-me/redux/currencyConversion';
 import logger from 'logger';
 
-export default function useInitializeWallet() {
+export default function useWalletManager() {
```
Comment (topwebtek7) [inline src/hooks/useWalletManager.ts:34]: nice refactor in this file! 👏🏼

File: src/hooks/useWalletManager.ts [hunk 2]
```diff
@@ -20,128 +27,142 @@ import { checkPushPermissionAndRegisterToken } from '@cardstack/models/firebase'
...
-        await dispatch(setCurrencyConversionRates(conversionsRates));
-
         logger.sentry('done loading network');
 
-        const isImporting = !!seedPhrase;
-        logger.sentry('isImporting?', isImporting);
-
         // TODO: move to fallbackExplorer, shouldn't be related with initializating a wallet
-        await loadCoingeckoCoins();
+        loadCoingeckoCoins();
```
Comment (topwebtek7) [inline src/hooks/useWalletManager.ts]: Agree with moving it into fallbackExplorer.. but till that move, can we keep it async call cause coins data is used in `initializeAccountData()` in the below this function and I think that's safer for a scenario that creating wallet first time.

**Review by topwebtek7 — APPROVED** [review]
Good job with nice refactoring! 🚀  just a small concern comment.

**Review by oidoug — APPROVED** [review]
I tested what I think these changes could impact and everything works well. Good job on the refactored wallet model :)

**Pull request merged**